pkg/signature/dsse/dsse_test.go

```diff
@@ -41,8 +41,8 @@ func TestRoundTrip(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	data := "sometestdata"
-	payloadType := "foo"
+	data := "\xb3"
```

Comment:
This is the failing case.

Reply:
@dlorenc FYI...looks like you have implemented the DSSE. It is failing for this data

Reply:
Do you have the failure handy? It might actually be in the upstream library (which I also implemented) over here: https://github.com/secure-systems-lab/dsse

Reply:
If the payloadType is set to `\xb3` it fails.